pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1432,3 +1441,65 @@ func decoratePciAddressField(addressField string) (*Address, error) {
 	}
 	return decoratedAddrField, nil
 }
+
+func createHostDevForGpu(c *ConverterContext) ([]HostDevice, error) {
+
+	var hostDevices []HostDevice
+	var parseErr error
+	gpuPciAddresses := append([]string{}, c.GpuDevices...)
+	_, isVgpu := os.LookupEnv("IS-VGPU")
```

Comment:
It looks that with this variable set only one type of devices can be attached to a VM.

Perhaps, it would be better to form the `HostDevices` by identifying the values in `c.GpuDevices` - if it's a uuid -> create mdev hostdev, if pciAddress for pci hostdev?
Alternatively, you could use two generic environment variables, one for GPU and other vGPU (GPU-PASSTHROUGH-DEVICES and VGPU-PASSTHROUGH-DEVICES)

Reply:
Implemented this change